apps/backend/routes.py

```
@@ -813,6 +816,237 @@ def delete_site(site_id: str, db: Session = Depends(get_db)):
+    )
+    if existing:
+        raise HTTPException(status_code=409, detail="This landing association already exists")
+
+    distance = haversine_distance(
+        takeoff.latitude, takeoff.longitude, landing.latitude, landing.longitude
+    )
+
+    if data.is_primary:
+        db.query(SiteLandingAssociation).filter(
+            SiteLandingAssociation.takeoff_site_id == site_id,
+            SiteLandingAssociation.is_primary == True,
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `True`; use `SiteLandingAssociation.is_primary:` for truth checks

Replace with `SiteLandingAssociation.is_primary`

(E712)